src/codegen_call_user.cpp

```
@@ -391,6 +391,7 @@ llvm::Value *CodeGen::emitUserFnCall(const std::string &callee, const std::vecto
 
 void CodeGen::emitStmt(CallStmt &s) {
```

Cppcheck (2.20.0)

[style] The function 'emitStmt' is never used.

(unusedFunction)